README.md

```diff
@@ -1,8 +1,54 @@
 # AEM Forms - Adaptive Forms
 
-Repo containing story book, js api docs and http api docs.
+AEM Forms now provide a mechanism to render Adaptive Form using React Components in Single Page Applications. It exposes headless APIs to fetch the 
+form definition from AEM and render it using JavaScript libraries.
```

Comment:
I think it should be beyond react components. It is more of marketing/introduction so would request PMs to pitch in.

Reply:
May be also say here or just after to try OOTB react super component using story book.